packages/manager/modules/bm-server-components/src/general-information/inputs/template.html

```diff
@@ -90,23 +90,12 @@ <h5 data-translate="server_configuration_installation_inputs_title"></h5>
         >
         </textarea>
         <!-- sshPubKey -->
-        <div
-            data-ng-switch-when="sshPubKey"
-            class="float-right sshkey-switch"
-            data-sshkey-switch
-            data-sshkey-switch-key="{{$ctrl.installation.input[input.name]}}"
-        ></div>
-        <textarea
-            data-ng-switch-when="sshPubKey"
-            class="oui-input form-control"
-            data-ng-model="$ctrl.installation.input[input.name]"
-            data-ng-required="input.mandatory"
-            rows="{{$ctrl.inputRules.sshPubKey.maxsize}}"
-            name="{{input.name}}"
```

Comment:
:warning: those questions are dynamic and depends on what the API returns. For the moment, question is named "sshKey" but it could change, especially if you have multiple SSH keys questions (https://stash.ovh.net/projects/DISTRIBPACK/repos/ostemplate/pull-requests/1339/diff#yaml%2Ftest-ux_64.yaml will be merged soon so you can see by yourself with "test-ux" OS)
Maybe you also need to make name an argument to that component, id as well I guess ?

Reply:
Hello JayBeeDe, Yes default value is already working. I will make these fields as parameter(rows, placeholder, name)